app/templates/base-cy.html

```diff
@@ -116,9 +116,9 @@
     {{
         onsCookiesBanner({
             'statementTitle': 'Dywedwch wrthym a ydych yn derbyn cwcis',
-            'statementText': "Rydym ni’n defnyddio <a href='" + domain_url_cy + "/cwcis" + "'>cwcis i gasglu gwybodaeth</a> am y ffordd rydych chi’n defnyddio cy.ons.gov.uk. Rydym ni’n defnyddio’r wybodaeth hon i sicrhau bod y wefan yn gweithio cystal â phosibl ac i wella ein gwasanaethau.",
-            'confirmationText': 'Rydych chi wedi derbyn yr holl gwcis. Gallwch chi <a href="' + domain_url_cy + '/cwcis' + '">newid eich dewisiadau o ran cwcis</a> ar unrhyw adeg.',
-            'secondaryButtonUrl': domain_url_cy + '/cwcis',
+            'statementText': "Rydym ni’n defnyddio <a href='" + domain_url_cy + "/cookies" + "'>cwcis i gasglu gwybodaeth</a> am y ffordd rydych chi’n defnyddio cy.ons.gov.uk. Rydym ni’n defnyddio’r wybodaeth hon i sicrhau bod y wefan yn gweithio cystal â phosibl ac i wella ein gwasanaethau.",
+            'confirmationText': 'Rydych chi wedi derbyn yr holl gwcis. Gallwch chi <a href="' + domain_url_cy + '/cookies' + '">newid eich dewisiadau o ran cwcis</a> ar unrhyw adeg.',
+            'secondaryButtonUrl': domain_url_cy + '/cookies',
```

Comment:
The link is wrong on the cookie banner. Should be `/cookies/`. Wonder if we can set it so it just doesn't matter 🤔